apps/src/weblab/WebLab.js

```diff
@@ -135,6 +142,18 @@ WebLab.prototype.init = function (config) {
     isProjectLevel: !!config.level.isProjectLevel,
   });
 
+  function onAddFileHTML() {
```

Comment:
We don't lint this, but we prefer function expressions `const onAddFileHTML = function () {` over function declarations [per airbnb](https://github.com/airbnb/javascript#functions), at least when they're not at the root of the module.